src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3485,15 +3485,15 @@ private static void maybeCreateVccPinAndPossibleInversion(SiteInst si, String si
         }
     }
 
-    //NOTE: SRL16E (reference name SRL16E, EDIFCell in RW) uses A2-A5, so we need to connect A1 & A6 to VCC,
-    //however, when SitePinInsts (e.g. A3) are already in GND, adding those again will cause problems to A1
     static HashSet<String> unisimFlipFlopTypes;
     static {
         unisimFlipFlopTypes = new HashSet<>();
         unisimFlipFlopTypes.add("FDSE");//S CE, logical cell
         unisimFlipFlopTypes.add("FDPE");//PRE CE
         unisimFlipFlopTypes.add("FDRE");//R and CE
         unisimFlipFlopTypes.add("FDCE");//CLR CE
+        unisimFlipFlopTypes.add("LDCE");
+        unisimFlipFlopTypes.add("LDPE");
```

Comment:
These are technically latches, so should the variable name and associated helper method name change respectively?